NPE generating change log for retrying-again

Got the following trying to generate change log for retrying-again:

```
Finding commits between v0.7.0..v1.0.0 in dir: /home/users/scottl/Projects/kiwiproject/retrying-again
Loading all commits between v0.7.0 and v1.0.0
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Collecting ticket ids from 5 commits.
Fetching ticket info from https://api.github.com/kiwiproject/retrying-again based on 3 ids [89, 88, 87]
Querying Github API for 3 tickets.
Request url: https://api.github.com/repos/kiwiproject/retrying-again/issues?page=1&per_page=100&state=closed&filter=all&direction=desc
Github API rate info => Remaining : 4996, Limit : 5000, Reset at: 2021-09-03T18:54:26
Next page 'Link' from Github: null
Exception in thread "main" java.lang.RuntimeException: Problems fetching 3 tickets from Github
	at org.kiwiproject.changelog.GithubTicketFetcher.fetchTickets(GithubTicketFetcher.kt:32)
	at org.kiwiproject.changelog.GenerateChangelog.generate(GenerateChangelog.kt:27)
	at MainKt.main(main.kt:25)
Caused by: java.lang.NullPointerException: null cannot be cast to non-null type kotlin.String
	at org.kiwiproject.changelog.GithubTicketFetcher.isIssueOrLonePr(GithubTicketFetcher.kt:75)
	at org.kiwiproject.changelog.GithubTicketFetcher.extractImprovements(GithubTicketFetcher.kt:65)
	at org.kiwiproject.changelog.GithubTicketFetcher.fetchTickets(GithubTicketFetcher.kt:29)
	... 2 more
```

--- a/src/main/kotlin/org/kiwiproject/changelog/GithubTicketFetcher.kt
+++ b/src/main/kotlin/org/kiwiproject/changelog/GithubTicketFetcher.kt
@@ -2,7 +2,9 @@ package org.kiwiproject.changelog
 
 import org.kiwiproject.changelog.config.ChangelogConfig
 import org.kiwiproject.changelog.config.GithubConfig
-import java.util.*
+import java.util.Collections
+import java.util.PriorityQueue
+import java.util.Queue
 
 class GithubTicketFetcher(
     githubConfig: GithubConfig,

